blob/blob.go

```diff
@@ -172,6 +173,7 @@ func (a *Attributes) As(i interface{}) bool {
 type Writer struct {
 	b          driver.Bucket
 	w          driver.Writer
+	tracer     *oc.Tracer
```

Comment:
`tctx` should probably move up here; it's currently in the section that says "These fields exist only when w is not yet created."

Reply:
I replaced both `tctx` and `tracer` with a closure.

Reply:
I did the same for Reader.